bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/SummernoteEditor.java

```diff
@@ -90,7 +90,7 @@ public void onComponentTagBody(MarkupStream markupStream, ComponentTag openTag)
     @Override
     public void renderHead(IHeaderResponse response) {
         response.render(CssHeaderItem.forReference(SummernoteEditorCssReference.instance()));
-        response.render(CssHeaderItem.forReference(FontAwesome5CssReference.instance()));
+        response.render(CssHeaderItem.forReference(FontAwesome6CssReference.instance()));
```

Comment:
I think we need `IBootstrapSettings#fontAwesomeCssReference` + getter/setter

Reply:
I did so according to my understanding.